### ISSUE TITLE
DropDown display value issue if filter is active

### ISSUE THREAD

**Issue by msdevcode** [radzenhq/radzen-blazor#643]
**Describe the bug**
DropDown display value issue if filter is active

**Additional context**
Im using two dropdowns with the same list of items (for example countries)

![Step1](https://user-images.githubusercontent.com/115212774/195559161-eabb716b-3767-4ce5-a291-3033a273c652.PNG)

![Step0](https://user-images.githubusercontent.com/115212774/195559106-4a9fc5da-9227-4628-a423-4b51d5ad004c.PNG)

If you choose a value from dropdown 2, dropdown 1 applies the value of dropdown 2

Labels on the right side will show the current value of the dropdown

**To Reproduce**

1. Use the filter of dropdown 1 and search for "Germany" and choose the result

![Step2](https://user-images.githubusercontent.com/115212774/195559394-801195d8-d300-4371-b75f-66e0847743da.PNG)

![Step3](https://user-images.githubusercontent.com/115212774/195559430-96fe2222-3ab7-4acb-b51d-568f9fb309ba.PNG)

2. Use dropdown 2 and choose a different value e.g. Latvia or Spain. Dropdown 1 applies the value correctly as you can see the labels on the right side but the display isnt showing the value

![Step4](https://user-images.githubusercontent.com/115212774/195559463-36f8c544-2d6f-4133-9b5f-318bb77c83d9.PNG)

4. Focus on dropdown 1 and you can see that the filter is still active with "Germany". Maybe this is the issue because the value is different to the active filter results. If you follow the steps without using any filter then its working fine.

![Step5](https://user-images.githubusercontent.com/115212774/195559489-642f1a36-81ac-4850-a989-f7fddf2fbee8.PNG)

**Expected behavior**
Display of Dropdown 1 should show the applied value

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Chrome
 - Version : 106.0.5249.103

**Comment by enchev** [comment]
Hey @msdevcode,

Without actual code demonstrating the issue we cannot do much.

**Comment by msdevcode** [comment]
Good evening @enchev,

here is my code of the sampleproject

https://github.com/msdevcode/radzenissuetest/blob/master/RadzenIssueTest/Pages/Index.razor

**Comment by enchev** [comment]
Hey @msdevcode,

Check carefully your code before submitting issues. Here is a working version:
![image](https://user-images.githubusercontent.com/5804953/195787446-76949778-ea48-43a5-994e-70805322929f.png)
![image](https://user-images.githubusercontent.com/5804953/195787352-adfec0cb-d9d0-4180-b4e1-5f0bce083b80.png)

**Comment by msdevcode** [comment]
Hi @enchev,

thanks for your reply and taking the time to check my code. I updated my sampleproject with your code. 

But unfortunately you missed my part with the OnChange-Method to apply the value of dropdown 2 to dropdown 1 and if you got the filter active at dropdown 1 the display dont show the value correctly.

![grafik](https://user-images.githubusercontent.com/115212774/195808957-93b0cd50-b72a-44fd-8510-098ef1594e38.png)

Steps to reproduce:
1. Use the filter at dropdown 1 and leave the control
![grafik](https://user-images.githubusercontent.com/115212774/195806987-d6d10f1c-d346-48c9-a786-7b38c37657c4.png)

2. Choose a value eg. latvia/spain at dropdown 2
![grafik](https://user-images.githubusercontent.com/115212774/195807443-408ea0ac-fdd0-4ff0-b10b-7b83c18806f6.png)

The value of dropdown 2 is applied to dropdown 1 but the display of dropdown 1 dont show the value correctly

Updated sampleproject code:
https://github.com/msdevcode/radzenissuetest/blob/master/RadzenIssueTest/Pages/Index.razor
